docs_src/OVMS/pipelinesetup.md

```diff
@@ -6,13 +6,11 @@
 git clone  https://github.com/intel-retail/automated-self-checkout.git && cd ./automated-self-checkout
 ```
 
-### Step 2: Install Golang 1.20
```

Comment:
I think we still need to keep the Install Golang 1.20 since profile launcher main is written in go.

Reply:
Antonio's PR for this issue https://github.com/intel-retail/automated-self-checkout/issues/293 remove that prerequisite, if just build, we don't need go. @vli11